base/deprecated.jl

```diff
@@ -1057,6 +1057,8 @@ for (Fun, func) in [(:IdFun, :identity),
         (::Type{typeof($(func))})() = $(func)
     end
 end
+@deprecate_binding CentralizedAbs2Fun typeof(centralizedabs2fun(0)).name.primary
```

Comment:
Can the type of the anonymous function instantiated in `centralizedabs2fun` somehow be determined without calling it with a dummy argument?